### ISSUE TITLE
Downloaded .zip file is a bit recursive

### ISSUE THREAD

**Issue by ReflexiveCode** [scaleworkspace/covid19#14]
When downloading the .zip file, several subfolders contain a .zip file with that folder's files. For example "Eye Shield V1" contains a "Eye Shield.zip" file that has another copy of "Eye Shield V1". Not sure if this is intentional--possibly for easy sharing?

**Comment by dkearingES** [comment]
Yes, I realize it's a bit redundant. The idea is for easier sharing. I can stop this in future versions.
